chanbackup/backup.go

```diff
@@ -18,8 +17,7 @@ type LiveChannelSource interface {
 
 	// FetchChannel attempts to locate a live channel identified by the
 	// passed chanPoint. Optionally an existing db tx can be supplied.
-	FetchChannel(tx kvdb.RTx, chanPoint wire.OutPoint) (
-		*channeldb.OpenChannel, error)
+	FetchChannel(chanPoint wire.OutPoint) (*channeldb.OpenChannel, error)
```

Comment:
nit (commit message): Looks like we're even just removing `kvdb.RTx` transactions, so no _writing_ involved in this commit (which makes this an easy change, would perhaps need to look into things more closely if we did _writes_ to two different DBs without having an over-arching transaction).

Reply:
cool - updated to say "reads" :)